component/issuers.libsonnet

```diff
@@ -86,9 +87,7 @@ local patchSpec(obj) = {
 
 // Consecutively apply patches to result of previous apply.
 local process(obj, initManifest) = std.foldl(
```

Comment:
I'd wrap this in a `std.prune()` so we keep the old behavior of pruning `null` values and other "empty" fields. This is probably not strictly necessary, but should make our lives easier in general.